client/assets/pages/_event_invite-page.scss

```diff
@@ -0,0 +1,46 @@
+.event-invite{
+    &__main_card{
+        box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
```

Comment:
Use hex codes for the colors. Another thing a are we supposed to use rems ? There's the rem function in `function.scss`.

Reply:
Sorry for the late feedback. I think the indentation should be 2 spaces

Reply:
on it @kenneth-kip